frontend/components/CalendarPicker.tsx

```diff
@@ -0,0 +1,71 @@
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { DateTimePicker, DateTimePickerProps } from "@mui/x-date-pickers/DateTimePicker";
+import { renderTimeViewClock } from "@mui/x-date-pickers/timeViewRenderers";
+import { ThemeHook } from "@/hooks/themeHook";
+import clsx from "clsx";
+import dayjs from "dayjs";
+
+interface CalendarDatePickerProps extends DateTimePickerProps<dayjs.Dayjs> {
+  onDateChange: (date: dayjs.Dayjs | null) => void;
+  onEnableChange: (enabled: boolean) => void;
+}
+
+export const CalendarPicker = (props: CalendarDatePickerProps) => {
+  const {
+    timezone = "system",
+    format = "MM/DD/YY hh:mm:ss a",
+    timeSteps = { minutes: 1, seconds: 5 },
+    disablePast = true,
+    disabled = false,
+    onDateChange,
+    onEnableChange,
+    value = dayjs(),
+  } = props;
+
+  const { theme } = ThemeHook();
+
+  function handleDateChange(date: dayjs.Dayjs | null) {
```

Comment:
If is function. goes on the bottom of the component.

Reply:
Done